integration-tests/templates/sdk-dependency/_Cargo_explicit.toml

```diff
@@ -7,12 +7,10 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
+near-sdk = { features = ["abi"], ::sdk-version-toml:: }
 serde = { version = "1", features = ["derive"] }
 schemars = "0.8"
 
-[dependencies.near-sdk]
-::sdk-version-toml-table::
-
 [workspace]
```

Comment:
This was actually meant to test the table variant of specifying the SDK dependency. `explicit` is probably a confusing name, sorry about that